.github/CODEOWNERS

```diff
@@ -0,0 +1,43 @@
+# This file contains the list of default owners for the repo.
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*                             @math-fehr @webmiche
+
+
+
+# xDSL core
+xdsl/*                        @math-fehr @webmiche
+xdsl/utils/**                 @math-fehr @superlopuh
+
+# xdsl-opt tool
+xdsl/tools/**                 @math-fehr @webmiche
+
+# xDSL dialects
+xdsl/dialects/*               @math-fehr @webmiche
+
+# xDSL RISCV dialects
+xdsl/dialects/riscv*          @AntonLydike @superlopuh @compor @nazavode
+
```

Comment:
```suggestion

# xDSL RISCV extension dialects
xdsl/dialects/snitch.py       @nazavode
xdsl/transforms/*snitch*      @nazavode

```

Adding myself for the currently available RISC-V extension dialects in xDSL along the lines of the MPI stuff @AntonLydike